addons/events/fnc_addPlayerEngineEvent.sqf

```diff
@@ -0,0 +1,70 @@
+#include "script_component.hpp"
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_addPlayerEngineEvent
```

Comment:
The function adds an Event **Handler** so the name should reflect that.

We have `addBISEventHandler` so something like `addBISPlayerEventHandler` would be fitting. I don't like how the capitalized letters merge though.

Reply:
anyone have a different name suggestion?
otherwise I think it's fine as is